backend/src/main/kotlin/app/ehrenamtskarte/backend/freinet/webservice/schema/types/XMLAgencies.kt

```diff
@@ -0,0 +1,12 @@
+package app.ehrenamtskarte.backend.freinet.webservice.schema.types
+
+import com.fasterxml.jackson.annotation.JsonProperty
+import com.fasterxml.jackson.dataformat.xml.annotation.JacksonXmlElementWrapper
+import com.fasterxml.jackson.dataformat.xml.annotation.JacksonXmlRootElement
+
+@JacksonXmlRootElement(localName = "agencies")
+data class XMLAgencies(
+    @JsonProperty("mandant")
+    @JacksonXmlElementWrapper(useWrapping = false)
+    var agencies: ArrayList<XMLAgency> = ArrayList()
```

Comment:
:question: Is there any reason this needs to be `var` instead of `val`?
```suggestion
    var agencies: ArrayList<XMLAgency> = ArrayList()
```

Reply:
no i think not but your suggested changes is wrong :)